ci_example_python/demos/demo_pid.py

```diff
@@ -0,0 +1,82 @@
+#! /usr/bin/python
+
+##@package ci_example_python
+# 
+# @file demo_pid.py
+# @author Vincent Berenz
+# @author Maximilien Naveau
+# license License BSD-3-Clause
+# @copyright Copyright (c) 2019, New York University and Max Planck Gesellschaft.
+# @date 2019-05-22
+# 
+# @brief This demos present the API of the PID class.
```

Comment:
Are those doxygen tags parsed here? If not, the more Pythonic way would be to use metadata-variables like `__author__`. See https://stackoverflow.com/a/1523456/2095383.

And the use of a docstring instead of comment would have the advantage that it can be displayed in Python with `help(modulename)`.

Reply:
I had trouble with doxygen parsing the classic doc string I will investigate more. I agree with you in theory, I hope in practice we can do that.

Reply:
Feels like this is not possible to have the special commands if you use the classic doc string...
http://doxygen.nl/manual/docblocks.html#pythonblocks
I will look for a way.